meson.build

```diff
@@ -23,8 +23,16 @@ config_h = configure_file(
   output: 'config.h',
 )
 
+drm_fourcc_header_path = join_paths(libdrm.get_pkgconfig_variable('includedir'), 'libdrm/drm_fourcc.h')
+pymod = import('python')
+py3 = pymod.find_installation('python3')
+
+tables_h = custom_target('tables_h',
+  output : 'tables.h',
+  command : [py3, join_paths(meson.source_root(), 'fourcc.py'), drm_fourcc_header_path, 'tables.h', '@OUTPUT@'])
```

Comment:
You don't need the `'tables.h'` part here. `'@OUTPUT@'` will handle that.

Reply:
If I remove `output : 'tables.h',` line I get `ERROR: Missing keyword argument "output".`

Reply:
No, you keep the `output: 'tables.h'` argument. Just remove it from the command array.
`@OUTPUT@` will automatically be replaced by it, and allows you to not repeat yourself in the definition.

Reply:
I see, that works